src/Sidesheet/index.js

```diff
@@ -141,15 +141,17 @@ const Sidesheet = ({
             <Box
               flex
               flexDirection="col"
+              overflow={"overflow-y-auto"}
```

Comment:
This prop isn't really supported by Box. This just removes the overflow tailwind prop from Box and its the one in className that actually works. To do this correctly, we need to add overflow-y-auto to the list of overflow options in Box and remove the overflow className here.

Reply:
Fixed!